README.md

```diff
@@ -2,17 +2,17 @@
 
 Flutter plugin to enable, disable or toggle screenshot support in your application.
 
-
 ## Features
-  - Disables screenshot and screen recoding on Android and iOS
-  - Enables screenshot and screen recoding on Android and iOS
-  - Toggle screenshot and screen recoding on Android and iOS 
-## Getting started
 
-If you want to prevent user from taking screenshot or recording of your app. You can turn off the screenshot support from the root `didChangeAppLifecycleState` method. 
+- Disables screenshot and screen recoding on Android and iOS
+- Enables screenshot and screen recoding on Android and iOS
+- Toggle screenshot and screen recoding on Android and iOS
```

Comment:
Consider adding the letter "s" to Toggle in order to make it consistent with the latter